packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -289,7 +291,11 @@ Future<Null> _buildGradleProjectV1(FlutterProject project, String gradle) async
   printStatus('Built ${fs.path.relative(project.android.gradleAppOutV1File.path)}.');
 }
 
-Future<Null> _buildGradleProjectV2(String gradle, BuildInfo buildInfo, String target) async {
+Future<Null> _buildGradleProjectV2(
+    FlutterProject flutterProject,
+    String gradle,
```

Comment:
This seems it should be derrivable from flutterProject  or a GradleProject
Not necessarily in this CL.